.github/workflows/create-release.yaml

```
@@ -85,29 +87,26 @@ jobs:
         with:
           fetch-depth: 0
 
-      - uses: autotelic/action-wait-for-status-check@6556cf50c8fb6608412945382eae73581f56cbb4
-        name: Build image
-        id: wait-for-build
+      - name: Image build
```

Should we use the same name we use on main?
```suggestion
      - name: Build manager image
```

---

looks better, let's do that